projects/assets-library/assets/styles/_layout.scss

```diff
@@ -36,10 +36,39 @@
   padding: 0;
 }
 
+@mixin hidden {
+  display: none !important;
+}
+
+@mixin list-hover {
+  &:hover {
+    background: $gray-1;
+    cursor: pointer;
+  }
+}
+
+@mixin card-hover {
```

Comment:
Do they really belong to layout? May be `_interaction.scss`?

Reply:
Yeah, I almost through about creating another file. These certainly didn't fit well in 'fonts', but only fit slightly better in 'layouts'. We can continue to divide these up into additional files as they grow.

Reply:
Can we do it now?  these mixins in layout are confusing to me.

Reply:
I will in a follow up. I have a few branding/styling tickets to follow this one. I'll change it in one of them.